### PR TITLE
Handle exceptions in Cleaner.clean_html

### PR DESCRIPTION
**Pull request by whalebot-helmsman**
We need this, until https://bugs.launchpad.net/lxml/+bug/1838497 is resolved

### CODE REVIEW

File: html_text/html_text.py [hunk 1]
```diff
@@ -39,7 +39,13 @@ def _cleaned_html_tree(html):
         tree = html
     else:
         tree = parse_html(html)
-    return cleaner.clean_html(tree)
+
+    try:
+        cleaned = cleaner.clean_html(tree)
+    except Exception:
```
Comment (lopuhin) [inline html_text/html_text.py]: Could you please add a reference to the bug report here?

File: html_text/html_text.py [hunk 2]
```diff
@@ -39,7 +39,13 @@ def _cleaned_html_tree(html):
...
+        cleaned = lxml.html.Element('html')
```
Comment (lopuhin) [inline html_text/html_text.py]: This is essentially an empty element, right? Another option would be to return ``tree`` unchanged - that would be less surprising for me, probably. Do you think returning an empty result is better in this base?

**Comment by Gallaecio** [comment]
- Should we log a warning or something?
- Can we catch something more concrete than `Exception`?

**Comment by whalebot-helmsman** [comment]
> Should we log a warning or something?

Reasonable approach for me is using `logging` module. But we don't use `logging` in this library. Is it has sense to add logging only for this case?

**Pull request merged**